makefile

```diff
@@ -11,23 +11,23 @@ pretty: node_modules
 	find agcaballero/portfolio/src/main -iname *.html -o -iname *.css | xargs $(PRETTIER) --write
 	find agcaballero/portfolio/src/main -iname *.java | xargs $(CLANG_FORMAT) -i
 	find agcaballero/portfolio/src/main -iname *.js | xargs $(CLANG_FORMAT) -i
-	find jelares/portfolio/src/main -iname *.html -o -iname *.css | xargs $(PRETTIER) --write
-	find jelares/portfolio/src/main -iname *.java | xargs $(CLANG_FORMAT) -i
-	find jelares/portfolio/src/main -iname *.js | xargs $(CLANG_FORMAT) -i
-	find udaykalra/portfolio/src/main -iname *.html -o -iname *.css | xargs $(PRETTIER) --write
-	find udaykalra/portfolio/src/main -iname *.java | xargs $(CLANG_FORMAT) -i
-	find udaykalra/portfolio/src/main -iname *.js | xargs $(CLANG_FORMAT) -i
+#	find jelares/portfolio/src/main -iname *.html -o -iname *.css | xargs $(PRETTIER) --write
+#	find jelares/portfolio/src/main -iname *.java | xargs $(CLANG_FORMAT) -i
```

Comment:
May as well just delete these instead of commenting out.

Reply:
I believe I removed this from the commit just because I didn't want it to affect the repo.

Reply:
Done